Toolkit.Samples/Toolkit.Samples/App.xaml.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Xamarin.Forms;
+using Xamarin.Toolkit.Samples.Views;
+
+namespace Toolkit.Samples
+{
+	public partial class App : Application
+	{
+		public App ()
+		{
+			InitializeComponent();
+
+			MainPage = new NavigationPage(new MainPage());
+        }
```

Comment:
Formatting must have not happened here

Reply:
I'll fix this one up. Once we put in stylecop it will help tons too :)